.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -46,8 +46,8 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17 ]
-        bwc_version: [ "1.3.2" ]
-        opensearch_version: [ "2.0.0-rc1-SNAPSHOT", "2.0.0-SNAPSHOT" ]
+        bwc_version: [ "2.0.0" ]
+        opensearch_version: [ "2.1.0-SNAPSHOT" ]
```

Comment:
For rolling upgrade tests, we will upgrade from the last minor version of previous major version to all minor versions of current major version.
Reference - https://quip-amazon.com/lzkEA6OVlmjv#temp:C:SFc6fa8c229d21a652b821c326b3

```
bwc_version: [ "1.3.2" ]
opensearch_version: [ "2.0.0-SNAPSHOT", "2.1.0-SNAPSHOT" ]

Reply:
ack